resources/user-agents/apps/itunes/itunes-1-2-to-4-7.json

```diff
@@ -0,0 +1,55 @@
+{
+  "division": "iTunes #MAJORVER#.#MINORVER#",
+  "versions": ["4.7", "4.6", "4.5", "4.4", "4.3", "4.2", "4.1", 4, 3, 2, "1.2"],
+  "sortIndex": 6360,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "iTunes #MAJORVER#.#MINORVER#",
+      "engine": "WebKit",
+      "device": "Macintosh",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "iTunes #MAJORVER#.#MINORVER#",
+        "Browser": "iTunes",
+        "Browser_Type": "Application",
+        "Browser_Maker": "Apple Inc",
+        "Alpha": "false",
+        "Beta": "false",
+        "Frames": "false",
+        "IFrames": "false",
+        "Tables": "false",
+        "Cookies": "false",
+        "JavaScript": "false",
+        "isSyndicationReader": "false",
+        "Crawler": "false",
+        "Version": "#MAJORVER#.#MINORVER#",
+        "MajorVer": "#MAJORVER#",
+        "MinorVer": "#MINORVER#"
+      },
+      "children": [
+        {
+          "match": "iTunes/#MAJORVER#.#MINORVER#* (#PLATFORM#)*",
+          "device": "Macintosh",
+          "platforms": [
+            "OSX_10_4", "OSX_10_3", "OSX_10_2", "OSX_10_1",
+            "OSX",
+            "MacPPC"
+          ]
+        },
+        {
+          "match": "iTunes/#MAJORVER#.#MINORVER#* (#PLATFORM#)*",
+          "device": "Windows Desktop",
+          "platforms": [
+            "Windows", "Windows_64"
+          ]
+        },
+        {
+          "match": "iTunes?#MAJORVER#.#MINORVER#*",
```

Comment:
I had to use the question mark here on these generic patterns because otherwise these patterns were catching the Downcast and PodCruncher agents in the tests, which look like this:

>iTunes/10.7 Downcast/5608
>iTunes/10.7 Downcast/2.8.14.1002
>iTunes/10.5.2 (PodCruncher 2.2)

The patterns for these look like this:

```
iTunes/* Downcast/2.9*
iTunes/* Downcast/2.8*
iTunes/* Downcast/*
iTunes/* (PodCruncher *
```

The asterisk after the slash would cause these patterns to lose priority to the version expanded iTunes pattern from these files, which would look something like this after expansion:

```
iTunes/4.7*
```

Which is longer before the first wildcard character.  This is related to the issues described in https://github.com/browscap/browscap-php/issues/179

The question mark reduces the priority on these. Perhaps not ideal (since it'll match other varieties of this, like `iTunes_4.8`, but without fixing the parser I don't think there's another way to deal with this.